### ISSUE TITLE
OSX Bundle needs a log window

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#23]

**Comment by drtrigon** [comment]
May be this should be a own issue as it targets all OSes, but I wanted to ask to include the log output into the main window instead of having a dedicated window? May be configurable by command line parameter?

I imagine the same output as in the log window appearing in the main window e.g. bottom left and after 5 secs fading out. Would that be a good idea?

**Comment by Meakk** [comment]
I agree @drtrigon and we already discussed about that on Discord. I'd like to try a small integrated window using imgui.

**Comment by mwestphal** [comment]
Would that be copy/pastable ?

**Comment by Meakk** [comment]
I think so yes, we can see an input selected in the screenshot here: https://raw.githubusercontent.com/wiki/ocornut/imgui/web/v167/v167-misc.png

**Comment by mwestphal** [comment]
moving to 2.2 unless you want to tackle imgui right away @Meakk

**Comment by mwestphal** [comment]
Lets tackle that with imGui

**Comment by mwestphal** [comment]
Fixed by #1748
